Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -1181,18 +1177,19 @@ private command __fetchPropertyCustomGetter pGetter, pObject, pProperty, @xPropA
    end if
 end __fetchPropertyCustomGetter
 
+# MDW 2019.10.02 [[ feature_fast_stack_introspection ]] added pProperties parameter for speed
 private on __fetchPropertyOfControl pProp, @xData
```

Comment:
should'nt there be a third parameter "pProperties" for "__fetchPropertyOfControl"?

Reply:
Yep - you're commenting on an earlier commit. Somehow my latest edits didn't get in, so I edited again and pushed the corrected file as 82d41bb.